jekyll/_cci2/configuration-reference.md

```diff
@@ -31,7 +31,7 @@ Key | Required | Type | Description
 docker | Y <sup>(1)</sup> | List | Options for [docker executor](#docker-executor)
 machine | Y <sup>(1)</sup> | Map | Options for [machine executor](#machine-executor)
 steps | Y | List | A list of [steps](#steps) to be performed
-working_directory | Y | String | What directory to run the steps in (default: depends on executor)
+working_directory | Y | String | What directory to run the steps in. Can be short-handed as `workDir`.
```

Comment:
Reading again - I think we should not mention `workDir` even as an option. It's old-style and will be removed at some point in the future. from Mahmood: "we will continue to support the deprecated `workDir` until we decide to sunset "old" syntax" < but I don't think we should include both versions in the docs.